recipes/_openstack_security.rb

```diff
@@ -24,9 +24,5 @@
       expect(user('openstack-user')).to have_home_directory('/home/openstack-user')
       expect(user('openstack-user')).to have_login_shell('/bin/bash')
     end
-
-    it 'does not have a password' do
-      expect(command('passwd -S openstack-user').stdout).to match(/Password locked/)
-    end
```

Comment:
I don't think that giving a default password to a user on a public image is a good thing.  I think a locked password, combined with sudo access and SSH should be just fine.